client/trpc.ts

```diff
@@ -1,15 +1,22 @@
 import { createTRPCProxyClient, httpBatchLink } from '@trpc/client';
 import type { AppRouter } from 'server/src/routes/trpcRouter';
+import AsyncStorage from '@react-native-async-storage/async-storage';
 import { api } from './constants/api';
 
 console.log('api', api);
 
+const getUser = async (key) => {
```

Comment:
Token instead of user i think, but let me know if there's a reason to store whole user